sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -121,7 +121,7 @@ export class ECertFileHandler extends ESDCFileHandler {
       await this.disbursementScheduleService.getECertInformationToBeSent(
         offeringIntensity,
       );
-    if (!disbursements.length) {
+    if (!disbursements) {
```

Comment:
Is this wrong because of the behavior of `mapFromRawAndEntities` that is returning `null` when there are no entities?
![image](https://user-images.githubusercontent.com/61259237/210650303-3d981d93-c7fb-4996-97f1-540d8b405ecb.png)

Reply:
yes, its `mapFromRawAndEntities` returns null